packages/stark-ui/src/modules/collapsible/components/collapsible.component.html

```diff
@@ -0,0 +1,21 @@
+<mat-expansion-panel class="stark-collapsible" 
+					 [ngClass]="{ 'spin' : options.spin }"
```

Comment:
Shouldn't the "spin" class be set on the `<mat-icon>` element instead? Or am I missing something?